tests/test_generator.py

```diff
@@ -150,16 +150,23 @@ def test_run(self):
         generator_obj = Generator("blog",
             "Article",
             ("title:charfield", "body:textfield"))
-        generator_obj.core_dir = self.core_folder
-        generator_obj.api_dir = self.api_folder
-        result = generator_obj.run()
-        assert result == print("🎉 Your RESTful Article api resource is ready 🎉")
+        result = self._extracted_from_test_run_6(
```

Comment:
Unused variable (Relate to build Failed) #6 #5

Reply:
I Just revert `test_run`  to the old version. [48bf7ca](https://github.com/Abdenasser/dr_scaffold/pull/5/commits/48bf7ca4ea399e4ef7fa7b84c6aa208ae8478652)